suse_migration_services/migration_config.py

```diff
@@ -43,10 +52,16 @@ def get_migration_product(self):
         The value returned is passed to the --product option in the
         zypper migration call
 
-        migration:
-          target:
-            product: product_name
+        migration_product: product_name
         """
-        migration = self.config_data.get('migration')
-        if migration and migration.get('target'):
-            return migration.get('target').get('product')
+        migration_target = self.config_data.get('migration_product', 'sles')
+        migration_product = self.target_products.get(migration_target)
+
+        if not migration_product:
+            message = ('The configured migration product {0}'
+                       'is not supported, aborting migration attempt'
+                       .format(migration_target))
```

Comment:
Can we go and use the dedent module here: Example:

```python
message = dedent('''
    The configured migration product {0} is not supported,
    aborting migration attempt
''')

log.error(message.format(migration_target))
```

Reply:
Fixed.